docs/getting_started/creating_an_admin.rst

```diff
@@ -83,7 +89,36 @@ two very simple ``Post`` and ``Tag`` entities will be used::
         {
             return $this->blogPosts;
         }
+        
+        /**
+         * Get id
+         *
+         * @return integer 
+         */
+        public function getId() {
```

Comment:
please put the `{` brackets on the next line

Reply:
@OskarStark Thank you sir.
I have done this thing